app/assets/images/icons/instagram.svg:Zone.Identifier

```diff
@@ -0,0 +1,3 @@
+[ZoneTransfer]
+ZoneId=3
+HostUrl=https://www.figma.com/

```

Comment:
As @jclopezdev mentions, do you need this file?

Reply:
Its a autogenerated file, if we delete it, it reapears